docs/Systems/Tailwind/ClassnameReferences/TextColor.stories.tsx

```diff
@@ -24,32 +26,35 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
   isReversed,
 }) => (
   <>
-    <StoryWrapper isReversed={isReversed}>
+    <StoryWrapper hasRowDivider isReversed={isReversed}>
```

Comment:
Just checking here that we weren't using the UtilityClassTemplate because its this story has an additional example column

Reply:
Yep correct. Good pickup.

Might be worth abstracting if we get more reference docs with an extra column, but it's just this one at the moment.

But if we ever did add more, I'd be more tempted to just drop the fourth column, and instead render the example text with a swappable background.

Reply:
I've ended up using the UtilityClassTemplate for this story as well - just moved both light and dark examples into one component. They sit next to each other in one column now.